Wireframe/css/style.css

```diff
@@ -0,0 +1,67 @@
+body{
+    font-family: sans-serif;
+    font-family: 'Libre Baskerville', serif;
+    margin: 20px 100px;
+    background-color: rgb(233, 237, 240);
+ 
+    
+}
+
+header {
+   margin-bottom: 50px;
+}
+
+h1 {
+    font-size: 5rem;
+    text-align: center;
+}
+.article1 {
```

Comment:
The requirements call on you to "Use the :first-child pseudo-class to style the first article so that it stands out from the others", maybe give this another try using that pseudo class instead of the article1 class.

Reply:
That's right. I gave another class to that. Thank you